connectors/byoei.py

```diff
@@ -133,6 +136,13 @@ async def _batch_bulk(self, operations):
         return res
 
     async def run(self):
+        """Creates batches of bulk calls given a queue of documents.
+
+        Exits when the document is "END_DOCS" or "FETCH_ERROR".
```

Comment:
```suggestion
        Exits when the operation is either "END_DOCS" or "FETCH_ERROR".
```

Shouldn't that be "operation"?

Reply:
well, technically the queue holds docs, except for that very specific item that is used as a signal.
Maybe we could use the word `item` instead, and say that items in the queue can be documents or on of those constants